RELEASE/scripts/autoscend/paths/casual.ash

```diff
@@ -234,7 +234,8 @@ boolean LX_acquireFamiliarLeprechaun()
 				}
 				else if(zone_available($location[The Spooky Forest]))	//spend 1 adv and one clover to get it instead.
 				{
-					retrieve_item(1, $item[11-Leaf Clover]);
+					//store the result to avoid an error being thrown
+					boolean result = retrieve_item(1, $item[11-Leaf Clover]);
```

Comment:
```suggestion
					//catch to avoid an error being thrown
					catch retrieve_item(1, $item[11-Leaf Clover]);
```

Reply:
casual by definition has mall access. so it likely just ends up mallbuying it
catch does the same as storing the result